pkg/sdn/plugin/subnets.go

```diff
@@ -95,11 +95,6 @@ func (master *OsdnMaster) deleteNode(nodeName string) error {
 	if err != nil {
 		return fmt.Errorf("Error fetching subnet for node %q for deletion: %v", nodeName, err)
 	}
-	_, ipnet, err := net.ParseCIDR(sub.Subnet)
-	if err != nil {
-		return fmt.Errorf("Error parsing subnet %q for node %q for deletion: %v", sub.Subnet, nodeName, err)
-	}
-	master.subnetAllocator.ReleaseNetwork(ipnet)
```

Comment:
I do not see rapid events causing anything other than the cluster cidr filling up. i.e. nodes deleting and being created so fast that hostsubnet release is slower than node subnet acquisition. At which point a node will eventually fail to get a subnet. Nothing disastrous and it is a rare circumstance anyway.